datasets/senti_lex/senti_lex.py

```diff
@@ -0,0 +1,226 @@
+    "ht",
+    "hu",
+    "hy",
+    "ia",
+    "id",
+    "io",
+    "is",
+    "it",
+    "ka",
+    "km",
+    "kn",
+    "ko",
+    "ku",
+    "ky",
+    "la",
+    "lb",
+    "lt",
+    "lv",
+    "mk",
+    "mr",
+    "ms",
+    "mt",
+    "nl",
+    "nn",
+    "no",
+    "pl",
+    "pt",
+    "rm",
+    "ro",
+    "ru",
+    "sk",
+    "sl",
+    "sq",
+    "sr",
+    "sv",
+    "sw",
+    "ta",
+    "te",
+    "th",
+    "tk",
+    "tl",
+    "tr",
+    "uk",
+    "ur",
+    "uz",
+    "vi",
+    "vo",
+    "wa",
+    "yi",
+    "zh",
+]
+
+
+class SentiLex(datasets.GeneratorBasedBuilder):
+    """Sentiment lexicons for 81 different languages"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+  You should download the dataset from https://www.kaggle.com/rtatman/sentiment-lexicons-for-81-languages
+  The webpage requires registration.
+  After downloading, please put the files in a dir of your choice,
+  which will be used as a manual_dir, e.g. `~/.manual_dirs/senti_lex`
+  SentiLex can then be loaded via:
+  `datasets.load_dataset("newsroom", data_dir="~/.manual_dirs/senti_lex")`.
+  """
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name=i,
+            version=datasets.Version("1.1.0"),
+            description=("Lexicon of positive and negative words for the " + i + " language"),
+        )
+        for i in LANGS
+    ]
+
+    def _info(self):
+
+        features = datasets.Features(
+            {
+                "word": datasets.Value("string"),
+                "sentiment": datasets.ClassLabel(
+                    names=[
+                        "negative",
+                        "positive",
+                    ]
+                ),
+            }
+        )
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        data_dir = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
+        if not os.path.exists(data_dir):
+            raise FileNotFoundError(
+                "{} does not exist. Make sure you insert a manual dir via `datasets.load_dataset('newsroom', data_dir=...)` that includes files unzipped from the reclor zip. Manual download instructions: {}".format(
+                    data_dir, self.manual_download_instructions
+                )
+            )
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepaths_dict": [
+                        [
+                            os.path.join(data_dir, "sentiment-lexicons/" + i)
+                            for i in os.listdir(data_dir + "/sentiment-lexicons")
```

Comment:
```suggestion
                            os.path.join(data_dir, "sentiment-lexicons", i)
                            for i in os.listdir(os.path.join(data_dir, "sentiment-lexicons"))
```

use os.path.join to avoid issues on windows (it uses backslashes as path separator)